frontend/src/hocs/withMapboxToken/withMapboxToken.tsx

```diff
@@ -33,6 +33,14 @@ interface State {
   isFetching: boolean
 }
 
+// We consume a Component that takes a "mapboxToken" prop, and create
+// a wrapped Component that takes a "sessionInfo" prop, and omits
+// the "mapboxToken" prop
+export type WrappedProps<P extends InjectedProps> = Omit<P, "mapboxToken"> & {
```

Comment:
This HOC got a bit more complex because of the sessionInfo change, so I pulled this type out to the top level and documented it.